app/models/photo.rb

```diff
@@ -67,10 +67,8 @@ def clear_empty_status_message
 
   def ownership_of_status_message
     message = StatusMessage.find_by_guid(self.status_message_guid)
-    if self.status_message_guid && message
-      self.diaspora_handle == message.diaspora_handle
-    else
-      true
+    if self.status_message_guid && message && self.diaspora_handle != message.diaspora_handle
```

Comment:
- Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.
- Style/RedundantSelf: Redundant `self` detected.